FCCee/IDEA/compact/IDEA_o1_v03/DriftChamber_o1_v02.xml

```diff
@@ -0,0 +1,137 @@
+<lccdd>
+  <!-- %%%%%%             Central Drift Chamber Basic Parameters                 %%%%%% -->
+  <!-- %%%%%%           based on the geometry version "IDEA231026"               %%%%%% -->
+  <define>
+  <!--  Drift Chamber parameters    -->
+  <!--      Gas and vesssel geometry parameters moved to DetDimensions file   -->
+
+  <!-- Position of guard wires   -->
+  <!--    _z0 := at z=0          -->
+  <!--    _zL2 := at z=DCH_Lhalf -->
+  <constant name="DCH_guard_inner_r_at_z0"         value=" DCH_inner_cyl_R + 4*mm "    />
+  <constant name="DCH_guard_outer_r_at_zL2"        value=" DCH_outer_cyl_R - 12.5*mm " />
+
+  <!-- Parametrization of number of cells ands layer/superlayer   -->
+  <constant name="DCH_ncell"                value=" 192 "    />
+  <constant name="DCH_ncell_increment"      value=" 48 "    />
+  <constant name="DCH_nsuperlayers"         value=" 14 "    />
+  <constant name="DCH_nlayersPerSuperlayer" value=" 8 "    />
+  <constant name="DCH_ncell_per_sector"     value=" 24 "   />
```

Comment:
Is this used/needed?

Reply:
Out of those parameters, the only one that is not used is `DCH_ncell_per_sector`. I can remove it

`DCH_nlayersPerSuperlayer` and `DCH_nsuperlayers` are used to calculate total number of layers and increment the number of cells in a given layer adding `DCH_ncell_increment` times the superlayer.